cssdb.settings.json

```diff
@@ -949,12 +1066,15 @@
     "description": "A media query to detect if the user has requested the system use a light or dark color theme",
     "specification": "https://www.w3.org/TR/mediaqueries-5/#prefers-color-scheme",
     "stage": 2,
-    "caniuse": "prefers-color-scheme",
     "browser_support": {},
     "docs": {
       "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme"
     },
     "example": "body {\n  background-color: white;\n  color: black;\n}\n\n@media (prefers-color-scheme: dark) {\n  body {\n    background-color: black;\n    color: white;\n  }\n}",
+    "mdn_path": [
+      "css['at-rules'].media['prefers-color-scheme']",
+      "css['at-rules'].media['prefers-color-scheme']['no-preference']"
```

Comment:
This is deprecated and was also removed from the PostCSS plugin and polyfill.

Reply:
You're right! I forgot about it!